src/core/lib/draw-layers.js

```diff
@@ -155,14 +155,6 @@ function drawLayersInViewport(gl, {
 
   // const {x, y, width, height} = deviceRect || [];
 
-  // TODO: Update all layers to use 'picking_uActive' (picking shader module)
-  // and then remove 'renderPickingBuffer' and 'pickingEnabled'.
-  const pickingUniforms = {
```

Comment:
Isn't there are risk we break external layers with this change? Should we leave these until major release?

Reply:
That is true, I will undo this change. But one thing to note, these uniforms are set in core flow (draw-layers), so we will see the warning from luma.gl about these uniforms being deprecated always (whether application is actually using these uniforms or not).

Reply:
Hmm. That is not good. Maybe disable the warning for now then.

Reply:
Ok, I will undo this change here and will open a luma.gl PR to remove the warning.